cdap-docs/tools/doc-cdap-default.py

```diff
@@ -101,6 +110,34 @@ def encode(value):
         else:
             return ''
 
+    @staticmethod
+    def _split_text(text, length=70, delimiter='.'):
+        # Splits text into blocks not exceeding length if possible
+        if len(text) > length and text.count(delimiter):
+            # Split text into multiple pieces
+            split = text.rfind('.', 0, length)
```

Comment:
Seems like the delimiter is not being used, but rather a '.' is being used here.
This is improper, right?